recipes/pymagewell/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set owner = "KCL-BMEIS" %}
```

Comment:
You don't need to define a variable that is used only once. Also, if you hardcode this info below, the URL become more readable and folks can click on it.